### PR TITLE
Added support for de-DE 

### PR DESCRIPTION
**Pull request by YoussefMahmod**

### CODE REVIEW

**Comment by YoussefMahmod** [comment]
Hello @bndw I think this will work correctly, test it and let me know if it works as expected or not.

**Comment by Andanan** [comment]
There is already #98. So either contribute to that or consider to include the changes that i suggested there.

Btw. why did the lock-files change?

**Comment by Andanan** [comment]
Ok, i just took the time to compare the translations more closely. It seems that you already included all of my suggestions :+1: 

@bndw This PR seems more complete than #98 and it is based on the current master, so i guess it would be a better fit for the merge.

File: README.md [hunk 1]
```diff
@@ -41,15 +41,16 @@ make fmt.write
...
+| Chinese    | [Baoyuantop](https://github.com/Baoyuantop)        |
+| Spanish    | [oscfdezdz](https://github.com/oscfdezdz)          |
+| Portuguese | [pedrorenan](https://github.com/pedrorenan)        |
+| Japanese   | [hatsu38](https://github.com/hatsu38)              |
+| Russian    | [Teraskull](https://github.com/Teraskull)          |
+| Ukrainian  | [Teraskull](https://github.com/Teraskull)          |
+| Dutch      | [wouterbrink](https://github.com/wouterbrink)      |
+| French     | [Divlo](https://github.com/Divlo)                  |
+| Turkish    | [Riza Ergun](https://github.com/rizaergun)         |
+| German     | [Youssef Mahmoud](https://github.com/YoussefMahmod)|
```
Comment (Andanan) [inline README.md:56]: In my opinion it should mention both @YoussefMahmod  and @devofthings as they both made nontrivial additions to the german translations.

```suggestion
| German     | [Youssef Mahmoud](https://github.com/YoussefMahmod)<br>[devofthings](https://github.com/devofthings) |
```
This suggestions is not enough of course as i can only edit one of the lines and that would mess up the formatting of the table

Comment (YoussefMahmod) [reply README.md]: I apologize for these mistakes, I am new to open sourcing and this was my second PR ever, so thanks for correct me and i hope you to accept my apologize.

File: src/i18n.js [hunk 2]
```diff
@@ -339,6 +339,33 @@ const resources = {
...
+        'Druck dir eine simple Karte mit deinen WLAN-Zugangsdaten aus. Klebe sie an deinen Kühlschrank, behalte sie in deinem Portemonnaie, etc.',
+      'desc.privacy':
+        'Deine Zugangsdaten werden niemals zum Server gesendet. Es gibt kein Tracking, Analytics, oder Fingerprinting auf dieser Website. Hier geht es zum',
+      'desc.source': 'Quellcode',
+      'wifi.login': 'WLAN-Zugangsdaten',
+      'wifi.name': 'WLAN-Netzwerkname (SSID)',
+      'wifi.name.placeholder': 'WLAN-Netzwerkname',
+      'wifi.password': 'Passwort',
+      'wifi.password.placeholder': 'Passwort',
+      'wifi.password.hide': 'Passwordfeld vor dem Drucken ausblenden',
```
Comment (xasx) [inline src/i18n.js:355]:
```suggestion
      'wifi.password.hide': 'Passwortfeld vor dem Drucken ausblenden',
```

File: src/i18n.js [hunk 3]
```diff
@@ -339,6 +339,33 @@ const resources = {
...
+      'wifi.password.encryption': 'Verschlüsselung',
+      'wifi.tip':
+        'Zeige mit der Kamera deines Handys auf den QR-Code, um automatisch eine Verbindung herzustellen',
```
Comment (xasx) [inline src/i18n.js:358]:
```suggestion
        'Richte die Kamera deines Smartphones auf den QR-Code, um automatisch eine Verbindung herzustellen',
```

**Review by xasx — COMMENTED** [review]
Minor stuff (Denglisch) you may want to include

**Comment by bndw** [comment]
Hi @YoussefMahmod , German has since landed in master. If you believe there are corrections, please open a new pull request with changes.

You can find the existing German translation here: https://github.com/bndw/wifi-card/blob/master/src/i18n.js#L266-L292

**Pull request closed without merging**